app/views/device_stories/index.js.erb

```diff
@@ -0,0 +1,3 @@
+$('#results_class').html("<%= j (render 'results', device_stories: @device_stories) %>")
```

Comment:
I don't think you need the parentheses here. If you want to keep them `j render('results', device_stories: @device_stories)` might read a little more nicely